x-pack/solutions/security/plugins/security_solution/public/flyout/document_details/shared/hooks/use_highlighted_fields.ts

```diff
@@ -45,6 +52,7 @@ export interface UseHighlightedFieldsResult {
 export const useHighlightedFields = ({
   dataFormattedForFieldBrowser,
   investigationFields,
+  type = 'all',
```

Comment:
You might not have to have a default value here, as the only place where you use `type` in this hook is where you call `getHighlightedFieldsToDisplay` and the value is already defaulted to `all` there...

Reply:
Does this contradict with the earlier comment about removing the `all`?  https://github.com/elastic/kibana/pull/216740/files#r2027440880

Reply:
Yes sorry! I think it should be safe to remove the most level one, which is `useHighlightedFields` (and not `getHighlightedFieldsToDisplay`). Makes sense?